module/icp/algs/sha1/sha1.c

```diff
@@ -84,7 +84,7 @@ static uint8_t PADDING[64] = { 0x80, /* all zeros */ };
  * ROTATE_LEFT rotates x left n bits.
  */
 
-#if	defined(__GNUC__) && defined(_LP64)
+#if	defined(__GNUC__) && defined(_LP64) && !defined(__amd64__)
```

Comment:
Adding a architecture check here doesn't feel like the best way to silence this warning.  What if instead you moved the block which defines `ROTATE_LEFT` unmodified in to the `#if/else` block where it's used.